app/controllers/account/dashboard_controller.rb

```diff
@@ -2,5 +2,7 @@ class Account::DashboardController < ApplicationController
   before_action :authenticate_user!
   layout 'account'
 
-  def index; end
+  def index
+    render locals: { chart_value: current_companies }
```

Comment:
why not just assign to a instance variable instead?

Reply:
agree with Roman. code is not good